guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -25,7 +25,7 @@ endif::[]
 .Before You Begin
 
 * Review and update your firewall configuration before upgrading your {ProjectServer}.
-For more information, see {InstallingServerDisconnectedDocURL}Ports_and_Firewalls_Requirements_satellite[Ports and Firewalls Requirements] in _{InstallingServerDisconnectedDocTitle}_.
+For more information, see {InstallingServerDisconnectedDocURL}Port_and_firewalls_requirements_satellite[Ports and firewalls requirements] in _{InstallingServerDisconnectedDocTitle}_.
```

Comment:
Please change the link text to "Port and firewall requirements". In this context, "port" and "firewall" are adjectives modifying "requirements". Adjectives do not have plural forms. Example: human resources (correct) vs humans resources. Lunch hour vs lunches hours. 

You also need to change the end of the URL, since you changed the name of the file.

Reply:
Fixed.